spec/influxdb/cases/querying_issue_7000_spec.rb

```diff
@@ -0,0 +1,89 @@
+# TODO: Update or remove once https://github.com/influxdata/influxdb/issues/7000 has been addressed
+#       - These tests only verify that multi-line JSON responses are parsed and merged correctly.
```

Comment:
Updated to reflect the purpose of the test in multi-line JSON in accordance with the resolution of this issue.